go-controller/pkg/cni/helper_linux.go

```diff
@@ -150,16 +168,32 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, macAd
 
 // PlatformSpecificCleanup deletes the OVS port
 func (pr *PodRequest) PlatformSpecificCleanup() error {
-	ifaceName := pr.SandboxID[:15]
-	ovsArgs := []string{
-		"del-port", "br-int", ifaceName,
+	networkName := pr.CNIConf.Name
+	var portList []string
+	if networkName == "ovn-kubernetes" {
```

Comment:
please use the same helper as before isDefaultInterface, or better, create a helper function for it, since it's used in multiple places

Reply:
done